host_os.py

```diff
@@ -31,12 +33,40 @@
     'set-env': setup_environment,
     'build-iso': build_iso,
 }
+PACKAGE_DEPENDENCIES_FILE_PATH = "rpm_requirements.txt"
+MISSING_PACKAGE_DEPENDENCIES_ERROR = 4
+
+def is_package_installed(package_name):
+    """
+    Checks if a RPM package is installed
+    """
+
+    cmd = "rpm -q %s" % package_name
+    try:
+        run_command(cmd, shell=True)
+    except SubprocessError as e:
+        # rpm returns 1 when search string is not found and other non-zero values
+        # if an error occurred
+        #pylint: disable=no-member
+        if e.returncode == 1:
+            return False
+        else:
+            raise
+
+    return True
 
 
 if __name__ == '__main__':
     CONF = config.setup_default_config()
     subcommand = CONF.get('default').get('subcommand')
 
+    package_dependencies = open(PACKAGE_DEPENDENCIES_FILE_PATH).readlines()
+    package_dependencies = [p.strip('\n') for p in package_dependencies]
+    missing_packages = [p for p in package_dependencies if not is_package_installed(p)]
+    if missing_packages:
+        print("Following packages should be installed before running this script: %s" % ", ".join(missing_packages))
+        sys.exit(MISSING_PACKAGE_DEPENDENCIES_ERROR)
+
```

Comment:
IMHO using full words are clear than abbreviations (eg "package" is clearer than "pkg") in variables names. It is a matter of taste.

Code now fits in 79 columns